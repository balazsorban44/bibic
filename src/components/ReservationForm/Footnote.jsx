import React from 'react'
import BANK from '../../assets/icons/bank.png'
import CASH from '../../assets/icons/cash.png'
import MASTERCARD from '../../assets/icons/mastercard.png'
import SZEP from '../../assets/icons/szep.jpg'
import VISA from '../../assets/icons/visa.png'

const Footnote = () => <span className="footnote">Elfogadott fizetési módok
  <ul className="payment-methods">
    <li>
      <h6>Készpénz</h6>
      <img
        alt="Készpénzes fizetés"
        src={CASH}
      />
    </li>
    <li>
      <h6>Előreutalás</h6>
      <img
        alt="Előreutalás"
        src={BANK}
      />
    </li>
    <li>
      <h6>Visa</h6>
      <img
        alt="Visa elfogadóhely"
        src={VISA}
      />
    </li>
    <li>
      <h6>Mastercard</h6>
      <img
        alt="Mastercard elfogadóhely"
        src={MASTERCARD}
      />
    </li>
    <li>
      <h6>OTP SZÉP</h6>
      <img
        alt="OTP SZÉP Kártya elfogadóhely"
        src={SZEP}
      />
    </li>
  </ul>
  FIGYELEM! Felhívjuk figyelmét, hogy a foglalás elküldése fizetési kötelezettséget nem von maga után. Foglalását először jóvá kell hagyjuk. A fizetés helyben, vagy átutalással történik. További részletekért kérdezhet a megjegyzésben, vagy írhat az alábbi címre:
  <a href="mailto:szallasfoglalas@bibicvendeghazak.hu">szallasfoglalas@bibicvendeghazak.hu</a>
</span>

export default Footnote