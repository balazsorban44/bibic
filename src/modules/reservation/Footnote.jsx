import React from "react"
import {useTranslation} from "react-i18next"


import BANK from "assets/icons/bank.png"
import CASH from "assets/icons/cash.png"
import MASTERCARD from "assets/icons/mastercard.png"
import SZEP from "assets/icons/szep.jpg"
import VISA from "assets/icons/visa.png"

const methods = [CASH, BANK, VISA, MASTERCARD, SZEP]
const Footnote = () => {
  const [t] = useTranslation("reservation")
  return (
    <span className="footnote">
      {t("payment-methods.title")}
      <ul className="payment-methods">
        {methods.map((method, index) =>
          <li key={index}>
            <img
              alt={t(`payment-methods.altOptions.${index}`)}
              src={method}
            />
            <h6>{t(`payment-methods.options.${index}`)}</h6>
          </li>
        )}
      </ul>
      {t("payment-methods.footnote")}
      <a href="mailto:szallasfoglalas&#64;bibicvendeghazak.hu">{t("reservation-email")}</a>
    </span>
  )
}

export default Footnote