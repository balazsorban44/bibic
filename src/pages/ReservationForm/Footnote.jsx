import React from 'react'
import BANK from '../../assets/icons/bank.png'
import CASH from '../../assets/icons/cash.png'
import MASTERCARD from '../../assets/icons/mastercard.png'
import SZEP from '../../assets/icons/szep.jpg'
import VISA from '../../assets/icons/visa.png'
import {useTranslation} from 'react-i18next'

const Footnote = () => {
  const [t] = useTranslation()
  return (
    <span className="footnote">{t("reservation.accepted-payment-methods")}
      <ul className="payment-methods">
        <li>
          <h6>{t("reservation.payment-methods.cash.title")}</h6>
          <img
            alt={t("reservation.payment-methods.cash.alt")}
            src={CASH}
          />
        </li>
        <li>
          <h6>{t("reservation.payment-methods.wire-transfer.title")}</h6>
          <img
            alt={t("reservation.payment-methods.wire-transfer.alt")}
            src={BANK}
          />
        </li>
        <li>
          <h6>Visa</h6>
          <img
            alt={t("reservation.payment-methods.visa.alt")}
            src={VISA}
          />
        </li>
        <li>
          <h6>Mastercard</h6>
          <img
            alt={t("reservation.payment-methods.mastercard.alt")}
            src={MASTERCARD}
          />
        </li>
        <li>
          <h6>OTP SZÃ‰P</h6>
          <img
            alt={t("reservation.payment-methods.otp.alt")}
            src={SZEP}
          />
        </li>
      </ul>
      {t("reservation.no-payment-warning")}
      <a href="mailto:szallasfoglalas&#64;bibicvendeghazak.hu">{t("reservation.reservation-email")}</a>
    </span>
  )
}

export default Footnote