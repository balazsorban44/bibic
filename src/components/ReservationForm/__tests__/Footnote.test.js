import Footnote from "../Footnote"


it("renders correctly", () => {
  const wrapper = shallow(<Footnote/>)
  expect(wrapper).toMatchSnapshot()

})