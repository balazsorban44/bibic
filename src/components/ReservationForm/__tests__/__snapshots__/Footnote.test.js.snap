// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footnote />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "Elfogadott fizetési módok",
        <ul
          className="payment-methods"
        >
          <li>
            <h6>
              Készpénz
            </h6>
            <img
              alt="Készpénzes fizetés"
              src="cash.png"
            />
          </li>
          <li>
            <h6>
              Előreutalás
            </h6>
            <img
              alt="Előreutalás"
              src="bank.png"
            />
          </li>
          <li>
            <h6>
              Visa
            </h6>
            <img
              alt="Visa elfogadóhely"
              src="visa.png"
            />
          </li>
          <li>
            <h6>
              Mastercard
            </h6>
            <img
              alt="Mastercard elfogadóhely"
              src="mastercard.png"
            />
          </li>
          <li>
            <h6>
              OTP SZÉP
            </h6>
            <img
              alt="OTP SZÉP Kártya elfogadóhely"
              src="szep.png"
            />
          </li>
        </ul>,
        "FIGYELEM! Felhívjuk figyelmét, hogy a foglalás elküldése fizetési kötelezettséget nem von maga után. Foglalását először jóvá kell hagyjuk. A fizetés helyben, vagy átutalással történik. További részletekért kérdezhet a megjegyzésben, vagy írhat az alábbi címre:",
        <a
          href="mailto:szallasfoglalas@bibicvendeghazak.hu"
        >
          szallasfoglalas@bibicvendeghazak.hu
        </a>,
      ],
      "className": "footnote",
    },
    "ref": null,
    "rendered": Array [
      "Elfogadott fizetési módok",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              <h6>
                Készpénz
              </h6>
              <img
                alt="Készpénzes fizetés"
                src="cash.png"
              />
            </li>,
            <li>
              <h6>
                Előreutalás
              </h6>
              <img
                alt="Előreutalás"
                src="bank.png"
              />
            </li>,
            <li>
              <h6>
                Visa
              </h6>
              <img
                alt="Visa elfogadóhely"
                src="visa.png"
              />
            </li>,
            <li>
              <h6>
                Mastercard
              </h6>
              <img
                alt="Mastercard elfogadóhely"
                src="mastercard.png"
              />
            </li>,
            <li>
              <h6>
                OTP SZÉP
              </h6>
              <img
                alt="OTP SZÉP Kártya elfogadóhely"
                src="szep.png"
              />
            </li>,
          ],
          "className": "payment-methods",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h6>
                  Készpénz
                </h6>,
                <img
                  alt="Készpénzes fizetés"
                  src="cash.png"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Készpénz",
                },
                "ref": null,
                "rendered": "Készpénz",
                "type": "h6",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Készpénzes fizetés",
                  "src": "cash.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h6>
                  Előreutalás
                </h6>,
                <img
                  alt="Előreutalás"
                  src="bank.png"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Előreutalás",
                },
                "ref": null,
                "rendered": "Előreutalás",
                "type": "h6",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Előreutalás",
                  "src": "bank.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h6>
                  Visa
                </h6>,
                <img
                  alt="Visa elfogadóhely"
                  src="visa.png"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Visa",
                },
                "ref": null,
                "rendered": "Visa",
                "type": "h6",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Visa elfogadóhely",
                  "src": "visa.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h6>
                  Mastercard
                </h6>,
                <img
                  alt="Mastercard elfogadóhely"
                  src="mastercard.png"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Mastercard",
                },
                "ref": null,
                "rendered": "Mastercard",
                "type": "h6",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Mastercard elfogadóhely",
                  "src": "mastercard.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h6>
                  OTP SZÉP
                </h6>,
                <img
                  alt="OTP SZÉP Kártya elfogadóhely"
                  src="szep.png"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "OTP SZÉP",
                },
                "ref": null,
                "rendered": "OTP SZÉP",
                "type": "h6",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "OTP SZÉP Kártya elfogadóhely",
                  "src": "szep.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
      "FIGYELEM! Felhívjuk figyelmét, hogy a foglalás elküldése fizetési kötelezettséget nem von maga után. Foglalását először jóvá kell hagyjuk. A fizetés helyben, vagy átutalással történik. További részletekért kérdezhet a megjegyzésben, vagy írhat az alábbi címre:",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "szallasfoglalas@bibicvendeghazak.hu",
          "href": "mailto:szallasfoglalas@bibicvendeghazak.hu",
        },
        "ref": null,
        "rendered": "szallasfoglalas@bibicvendeghazak.hu",
        "type": "a",
      },
    ],
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Elfogadott fizetési módok",
          <ul
            className="payment-methods"
          >
            <li>
              <h6>
                Készpénz
              </h6>
              <img
                alt="Készpénzes fizetés"
                src="cash.png"
              />
            </li>
            <li>
              <h6>
                Előreutalás
              </h6>
              <img
                alt="Előreutalás"
                src="bank.png"
              />
            </li>
            <li>
              <h6>
                Visa
              </h6>
              <img
                alt="Visa elfogadóhely"
                src="visa.png"
              />
            </li>
            <li>
              <h6>
                Mastercard
              </h6>
              <img
                alt="Mastercard elfogadóhely"
                src="mastercard.png"
              />
            </li>
            <li>
              <h6>
                OTP SZÉP
              </h6>
              <img
                alt="OTP SZÉP Kártya elfogadóhely"
                src="szep.png"
              />
            </li>
          </ul>,
          "FIGYELEM! Felhívjuk figyelmét, hogy a foglalás elküldése fizetési kötelezettséget nem von maga után. Foglalását először jóvá kell hagyjuk. A fizetés helyben, vagy átutalással történik. További részletekért kérdezhet a megjegyzésben, vagy írhat az alábbi címre:",
          <a
            href="mailto:szallasfoglalas@bibicvendeghazak.hu"
          >
            szallasfoglalas@bibicvendeghazak.hu
          </a>,
        ],
        "className": "footnote",
      },
      "ref": null,
      "rendered": Array [
        "Elfogadott fizetési módok",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                <h6>
                  Készpénz
                </h6>
                <img
                  alt="Készpénzes fizetés"
                  src="cash.png"
                />
              </li>,
              <li>
                <h6>
                  Előreutalás
                </h6>
                <img
                  alt="Előreutalás"
                  src="bank.png"
                />
              </li>,
              <li>
                <h6>
                  Visa
                </h6>
                <img
                  alt="Visa elfogadóhely"
                  src="visa.png"
                />
              </li>,
              <li>
                <h6>
                  Mastercard
                </h6>
                <img
                  alt="Mastercard elfogadóhely"
                  src="mastercard.png"
                />
              </li>,
              <li>
                <h6>
                  OTP SZÉP
                </h6>
                <img
                  alt="OTP SZÉP Kártya elfogadóhely"
                  src="szep.png"
                />
              </li>,
            ],
            "className": "payment-methods",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h6>
                    Készpénz
                  </h6>,
                  <img
                    alt="Készpénzes fizetés"
                    src="cash.png"
                  />,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Készpénz",
                  },
                  "ref": null,
                  "rendered": "Készpénz",
                  "type": "h6",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Készpénzes fizetés",
                    "src": "cash.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h6>
                    Előreutalás
                  </h6>,
                  <img
                    alt="Előreutalás"
                    src="bank.png"
                  />,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Előreutalás",
                  },
                  "ref": null,
                  "rendered": "Előreutalás",
                  "type": "h6",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Előreutalás",
                    "src": "bank.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h6>
                    Visa
                  </h6>,
                  <img
                    alt="Visa elfogadóhely"
                    src="visa.png"
                  />,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Visa",
                  },
                  "ref": null,
                  "rendered": "Visa",
                  "type": "h6",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Visa elfogadóhely",
                    "src": "visa.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h6>
                    Mastercard
                  </h6>,
                  <img
                    alt="Mastercard elfogadóhely"
                    src="mastercard.png"
                  />,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Mastercard",
                  },
                  "ref": null,
                  "rendered": "Mastercard",
                  "type": "h6",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Mastercard elfogadóhely",
                    "src": "mastercard.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h6>
                    OTP SZÉP
                  </h6>,
                  <img
                    alt="OTP SZÉP Kártya elfogadóhely"
                    src="szep.png"
                  />,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "OTP SZÉP",
                  },
                  "ref": null,
                  "rendered": "OTP SZÉP",
                  "type": "h6",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "OTP SZÉP Kártya elfogadóhely",
                    "src": "szep.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
        "FIGYELEM! Felhívjuk figyelmét, hogy a foglalás elküldése fizetési kötelezettséget nem von maga után. Foglalását először jóvá kell hagyjuk. A fizetés helyben, vagy átutalással történik. További részletekért kérdezhet a megjegyzésben, vagy írhat az alábbi címre:",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "szallasfoglalas@bibicvendeghazak.hu",
            "href": "mailto:szallasfoglalas@bibicvendeghazak.hu",
          },
          "ref": null,
          "rendered": "szallasfoglalas@bibicvendeghazak.hu",
          "type": "a",
        },
      ],
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
